test/transformextract.test.ts

```diff
@@ -106,16 +103,16 @@ describe('extractTransforms()', () => {
     'trellis_barley.vl.json',
     'trellis_barley_independent.vl.json',
     'trellis_column_year.vl.json',
-    'trellis_cross_sort_array.vl.json',
-    'trellis_cross_sort.vl.json',
     'trellis_line_quarter.vl.json',
     'vconcat_weather.vl.json',
     'window_top_k_others.vl.json'
   ]);
   for (const file of fs.readdirSync(specsDir)) {
+    // If true make sure that specs in the failsList do indeed fail
+    const TEST_EXPECTED_FAILS = false;
     const filepath = specsDir + file;
-    if (filepath.slice(-5) === '.json' && !failsList.has(file)) {
-      it(`should compile ${filepath} to the same spec`, () => {
+    if (filepath.slice(-5) === '.json' && (!failsList.has(file) || TEST_EXPECTED_FAILS)) {
```

Comment:
I added back the ability for these tests to also check to make sure that items in the failing list do actually fail because it makes it a lot easier to see what specs are positively impacted by changes. This feature can be turned on or off via the TEST_EXPECTED_FAILS boolean.

I'd be fine with removing this if it's seen as unnecessary.

Reply:
When did we remove this? I think it makes a lot of sense to fail when new specs are passing so that we keep the list of failing examples up to date. I'd even remove the boolean.

Reply:
Looks like it was removed by this commit **https://github.com/vega/vega-lite/commit/4bebfe9c3c62cc60c3d1bda7290e2067f25f2f37#diff-4fecb77be8d42e352cbe3327b4b49aaeR116**